getting-started/deployment/aws-ec2.mdx

```diff
@@ -89,10 +89,71 @@ sudo docker compose down && sudo docker compose up -d
 
 ## TLS
 
-For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance.
+For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance, as shown in the next section on custom domains.
 
 Without TLS, cloud templates may not load properly when using the Safari browser.
 
+## Accessing OpenOps via a custom domain
+
+If you want to access your OpenOps instance via a custom domain (e.g., `https://openops.your-company.com`) instead of its public IP address, you need to set up an AWS Application Load Balancer (ALB) with a valid TLS certificate from AWS Certificate Manager (ACM), then create a new DNS record and update your OpenOps configuration.
+
```

Comment:
The instructions are guiding the user to set up TLS (SSL) and not just a custom domain, it's worth pointing that out. If it was only about a custom domain, without TLS, a simple DNS record would've been enough, without ALB or a certificate.

Reply:
Thanks, good point. Considering that it may be useful for some to set up TLS without configuring a custom domain, I've split this into two separate procedures. Check out the latest updates.